src/datasets/providers/env-var.ts

```diff
@@ -0,0 +1,27 @@
+import EnvVarSvg from '@site/static/img/env-var-no-fill.svg';
+import { Provider } from '.';
+
+export const EnvVar: Provider = {
+  name: 'Environment Variable ',
+  logo: EnvVarSvg,
```

Comment:
```suggestion
  logo: EnvVarSvg,
  excludeFromLandingPage: true,
```

This will avoid adding the logo to the landing page.

Reply:
Perfect, excluded from landing!